amida-tech/recruitment-registry
# Bug/rr 890 create smtp seed

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do?
- Removes the `create-smtp.js` file that was doing nothing

#### Related JIRA tickets:
- https://jira.amida-tech.com/browse/RR-890

#### How should this be manually tested?
- N/A... or run the `mocha` tests and see if they all still pass (they do for me)

#### Background/Context
- I believe this was introduced by Kevin when he was implementing backend changes for the SMTP resolver layer that allows RecReg to issue emails, perhaps with the intention of it being a seed file or perhaps to simply serve as documentation-by-code of what the app expects to have available in the database before issuing its first email. Either way, I'm of the mindset that we use fallback-environment variables for the `smtp` seed (because it contains an email/password combo) -- but this can be a task for later.

#### Screenshots (if appropriate):
